pinc/project_edit.inc

```diff
@@ -58,8 +58,8 @@ function check_user_can_load_projects($exit_if_not)
     global $site_manager_email_addr;
     if (user_has_project_loads_disabled()) {
         echo "
-              <div class='display-flex'><div class='callout'>"
-            ._("You are not currently permitted to create new projects, or move projects out of the unavailable state.")
+              <div class='display-flex' style='margin-bottom:0.5em'><div class='callout'>"
```

Comment:
@cpeel -- I'm not sure this is the correct way to do this, but it was the easiest. Currently, the bottom border of the call-out box has no margin between it and the border of the button below it (at least not in FF):
<img width="342" alt="image" src="https://user-images.githubusercontent.com/36800239/213841064-706fa830-d5ba-4b69-827a-0657ea0587fe.png">

No extra margin was needed on top, and I didn't want to mess with the styling on the callout box. I'd be happy to change this if you can point me in the right direction.

Reply:
I think this is a great fix to that problem 👍